lib/deleteRuleConfigKey.js

```diff
@@ -0,0 +1,9 @@
+const apiCall = require('./api');
+
+const deleteRuleConfigKey = async () =>
+  apiCall({
+    path: 'rules-configs/AUTH0_ACCOUNT_LINKING_EXTENSION_CUSTOM_DOMAIN',
```

Comment:
nit: I'd put the key as a method parameter or give this method a more specific name

Reply:
Going to update it to 
```
const updateRuleConfigKey = async ({ path, method, ...options }) =>
  apiCall({
    path,
    method,
    ...options
  });
```
and reuse it for set and delete. Passing in the options as parameters

Reply:
```
    await updateRuleConfigKey({
      path: `rules-configs/${SETTINGS_CONFIG_KEY}`,
      method: 'PUT',
      body: { value: registeredCustomDomain }
    });
``` 
and
```
      await updateRuleConfigKey({
        path: `rules-configs/${SETTINGS_CONFIG_KEY}`,
        method: 'DELETE'
      });
```